### PR TITLE
Added LunaSA DTs for Security

### PR DESCRIPTION
**Pull request by millevy**
This DTs are for the testing of LunaSA HA HSM and LunaSA HSM.

### CODE REVIEW

**Comment by tosky** [comment]
Shouldn't these better addressed at the component level, as the testing is all focused on barbican (ok, with a few more components, but for a limited amount of tests) then here as DT?

File: examples/dt/barbican_LunaSA_HA_HSM_DT.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+| Revision | Change                | Date              |
+|--------: | :-------------------- | :--------------:  |
+| v0.1     | Initial publication   | %13/12/2023%      |
+
+## Node topology
+| Node role                                     | bm/vm    | amount |
+| --------------------------------------------- | -----    | ------ |
+| Openshift master/worker combo-node cluster    | vm       | 3      |
+| Compute nodes                                 | vm       | 1 or > |
+| LunaSA HSM                                    | RDU2/bm  | 2      |
```
Comment (eshulman2) [inline examples/dt/barbican_LunaSA_HA_HSM_DT.md:16]: do we have this HW only in rdu?

**Comment by millevy** [comment]
Due to change in priority, this DT is marked as draft.

**Comment by fultonj** [comment]
Closing due to inactivity. It can be re-opened if necessary.

**Pull request closed without merging**